packages/expo-asset/ios/AssetModule.swift

```diff
@@ -20,6 +20,7 @@ public class AssetModule: Module {
     AsyncFunction("downloadAsync") { (url: URL, md5Hash: String?, type: String, promise: Promise) in
       if url.isFileURL {
         promise.resolve(url.standardizedFileURL.absoluteString)
```

Comment:
Maybe resolve/reject should have the return type `Never` so it basically requires to be the last expression within the block? 🤔

Reply:
> Maybe resolve/reject should have the return type `Never` so it basically requires to be the last expression within the block? 🤔

i'll leave it to you and merge the pr first for fixing ci. thanks!